tests/unit_tests/sub_classing_test.py

```diff
@@ -0,0 +1,114 @@
+import unittest
+import numpy as np
+
+from super_gradients.training.datasets import DetectionDataSetV2
+from super_gradients.training.utils.detection_utils import DetectionTargetsFormat
+from super_gradients.training.exceptions.dataset_exceptions import EmptyDatasetException
+
+
+class TestDetectionDataSet(DetectionDataSetV2):
+    def __init__(self, input_dim, *args, **kwargs):
+        """Dummy Dataset testing subclassing, designed with no annotation that includes class_2."""
+
+        self.DUMMY_TARGETS = [np.array([[0, 0, 10, 10, 0],
```

Comment:
Again, what's with all these uppercase letters ? Where did you see we do this ?

Reply:
Same, changed in d5bc0d4b6a901e2db163f8fa85597a762e1142eb